assets/js/state/sapSystems.js

```diff
@@ -223,10 +243,12 @@ export const {
   removeTagFromSAPSystem,
   removeSAPSystem,
   updateSAPSystem,
+  updateApplicationInstanceAbsentAt,
   setApplicationInstanceDeregistering,
   unsetApplicationInstanceDeregistering,
   setDatabaseInstanceDeregisteringToSAPSystem,
   unsetDatabaseInstanceDeregisteringToSAPSystem,
+  updateDatabaseInstanceAbsentToSAPSystem,
```

Comment:
I would put this together with the first one, in line 247, as they are related

Reply:
done